components/searchInput.vue

```diff
@@ -174,15 +178,34 @@ export default {
     },
     students() {
       this.studentsData = [...this.students]
-    }
-  },
-
-  afterMounted() {
-    if (this.students.length < 1 && this.defaultFilters.isActive === 1) {
-      this.$toast.open({
-        message: 'Não há alunos ativos neste curso',
-        type: 'is-danger'
-      })
+    },
+    studentsData() {
```

Comment:
Você tem uma função com o mesmo nome de uma propriedade.

this.studentsData é uma array ou uma função? Melhor evitar ambiguidades.

Reply:
O método studentsData não tem um nome descritivo

Reply:
Isso era um watch, logo o método deve ter o mesmo nome da variável stalkeada, mas já troquei para computed (tableData) como foi mencionado antes. Creio que agora esteja dentro dos conformes.